cmd/otel-allocator/watcher/promOperator.go

```diff
@@ -171,7 +185,18 @@ func (w *PrometheusCRWatcher) LoadConfig() (*promconfig.Config, error) {
 			},
 		},
 	}
-	generatedConfig, err := w.configGenerator.Generate(prom, serviceMonitorInstances, podMonitorInstances, map[string]*monitoringv1.Probe{}, &store, nil, nil, nil, []string{})
+
+	generatedConfig, err := w.configGenerator.Generate(
+		prom,
+		serviceMonitorInstances,
+		podMonitorInstances,
+		map[string]*monitoringv1.Probe{},
+		store,
+		nil,
+		nil,
+		nil,
```

Comment:
Just for posterity, could you include what _could_ be set here? (in case someone in the future asks why we can't do X with service monitors it makes it more discoverable)

Reply:
Sure thing, I can add some TODO comments 👍